lib/spack/spack/modules.py

```diff
@@ -327,6 +327,9 @@ def blacklisted(self):
         blacklist_matches = [x
                              for x in configuration.get('blacklist', [])
                              if self.spec.satisfies(x)]
+        blacklisted_as_implicit = (configuration.get('blacklist_implicits') and
```

Comment:
@hartzell If you have better names feel free to use them, this was just a suggestion :grin: